application/wso2verify/app/google-services.json

```diff
@@ -1,34 +1,34 @@
 {
   "project_info": {
-    "project_number": "442076540514",
-    "firebase_url": "https://wso2-verify.firebaseio.com",
-    "project_id": "wso2-verify",
-    "storage_bucket": "wso2-verify.appspot.com"
+    "project_number": "1088404967999",
+    "firebase_url": "https://wso2-verify-b1a35.firebaseio.com",
+    "project_id": "wso2-verify-b1a35",
+    "storage_bucket": "wso2-verify-b1a35.appspot.com"
   },
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "1:442076540514:android:0637ab81732a744b1fab3b",
+        "mobilesdk_app_id": "1:1088404967999:android:11fd74887d59a014b39e8a",
         "android_client_info": {
           "package_name": "org.wso2.carbon.identity.mobile.wso2verify"
         }
       },
       "oauth_client": [
         {
-          "client_id": "442076540514-boagcuuri18gk0rib1i6lcsanjkiom8s.apps.googleusercontent.com",
+          "client_id": "1088404967999-0ck8mm5vqtdr1l3airk0juhibkjph7os.apps.googleusercontent.com",
           "client_type": 3
         }
       ],
       "api_key": [
         {
-          "current_key": "AIzaSyC4NM7UGzng3_JgiCoGhwtN2izx8wvMiaY"
+          "current_key": "AIzaSyA018Kboj82dRP9wZ9gozSEpPL15jqWxsg"
```

Comment:
Add dummy values here and in any other credential

Reply:
Leave the key empty and add to the ReadMe on how to get the key